core-bundle/assets/controllers/choices-controller.js

```diff
@@ -1,14 +1,16 @@
 import {Controller} from "@hotwired/stimulus"
 
 export default class ChoicesController extends Controller {
+    mutationGuard = false;
+
     connect() {
-        if (this.initGuard) {
+        if (this._isGuarded()) {
```

Comment:
We could try if removing/re-adding the `data-controller` attribute during init and destroy would also work (instead of the guard).

Reply:
Would that have any advantages?

Reply:
I thought this would remove the duplicate calls in the dev console. But readding the attribute will also trigger a connect, so forget about this idea. :slightly_smiling_face: